packages/host/testem.js

```diff
@@ -31,19 +32,27 @@ const config = {
 };
 
 if (process.env.CI) {
-  fs.mkdirSync('../../junit');
+  const junitDir = path.join(__dirname, '..', '..', 'junit');
+  fs.mkdirSync(junitDir, { recursive: true });
+  const testemLog = fs.createWriteStream(
+    path.join(junitDir, 'host-testem.log'),
+  );
 
   const reporters = [
     {
       ReporterClass: TapReporter,
       args: [false, null, { get: () => false }],
     },
+    {
+      ReporterClass: TapReporter,
+      args: [false, testemLog, { get: () => false }],
+    },
```

Comment:
Is there an advantage to doing it this way vs hoisting it to happen in the CI script like [this](https://github.com/cardstack/boxel/blob/ac551ef65d5e1daa8cb3e98985d670a6e9f9ffc4/.github/workflows/ci-host.yaml#L63-L65)?